client/web/src/components/users/List/__snapshots__/UserListRow.spec.tsx.snap

```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
What is this file? is it wanted ?

Reply:
it is a snapshot of component, it is easier mean to test a component